contrib/lax_der_parsing.c

```diff
@@ -51,7 +51,7 @@ int ecdsa_signature_parse_der_lax(const secp256k1_context* ctx, secp256k1_ecdsa_
     lenbyte = input[pos++];
     if (lenbyte & 0x80) {
         lenbyte -= 0x80;
-        if (pos + lenbyte > inputlen) {
+        if (lenbyte > inputlen - pos) {
```

Comment:
`pos` could possibly be larger than `inputlen` here, since it was just post-incremented on line 51 without a check.

Reply:
Right before line 51, we have `pos < inputlen`  since we have `pos != input` due to the check in line 48.
So after the increment in line 51, we have `pos <= inputlen`